sympy/geometry/parabola.py

```diff
@@ -376,6 +376,22 @@ def p_parameter(self):
             p = sign(self.focus.args[1] + y)
         return p * self.focal_length
 
+    @property
+    def latus_rectum(self):
+        """Returns the line passing through the focus
+           that is perpendicular to the axis of symmetry
+
+           Examples
+           ========
+
+           >>> p1 = Parabola(Point(0, 0), Line(Point(5, 8), Point(7, 8)))
+           >>> p1.latus_rectum
+           Line2D(Point2D(0, 0), Point2D(1, 0))
+
```

Comment:
It is failing doctest as you have not imported required functions/classes.
```suggestion

           >>> from sympy import Point, Line, Parabola
           >>> p1 = Parabola(Point(0, 0), Line(Point(5, 8), Point(7, 8)))
           >>> p1.latus_rectum
           Line2D(Point2D(0, 0), Point2D(1, 0))

```